cms/static/js/views/utils/view_utils.js

```diff
@@ -186,6 +214,7 @@ define(["jquery", "underscore", "gettext", "js/views/feedback_notification", "js
             'setScrollOffset': setScrollOffset,
             'redirect': redirect,
             'reload': reload,
-            'hasChangedAttributes': hasChangedAttributes
+            'hasChangedAttributes': hasChangedAttributes,
+            'keywordValidator': keywordValidator,
```

Comment:
Sadly, JavaScript only (relatively) recently added support for trailing commas in object/array definitions; this causes syntax errors in certain browsers/runtimes.

We'll get a clean diff if you move `keywordValidator` above `hasChangedAttributes` and remove the trailing comma from the `hasChangedAttributes` line.